docs/overview/getting-started/avalanche-platform.md

```diff
@@ -4,40 +4,53 @@ description: Avalanche features 3 built-in blockchains which includes Exchange C
 
 # Avalanche Platform
```

Comment:
I think we should call out some Subnets on this page. Level 1 header primary network with the three chains, then another level 1 header with Subnets and list DFK, Crabada, and more.

Reply:
I feel like this would better belong under the subnets/ folder instead of the intro/getting started page. For now I think we can do this in a future PR